app/models/bank_account.rb

```diff
@@ -27,12 +29,23 @@
 class BankAccount < ApplicationRecord
   belongs_to :intake
   has_one :client, through: :intake
-  attr_encrypted :bank_name, key: ->(_) { EnvironmentCredentials.dig(:db_encryption_key) }
-  attr_encrypted :account_number, key: ->(_) { EnvironmentCredentials.dig(:db_encryption_key) }
-  # Enum values are acceptable BankAccountType values to be sent to the IRS (See efileTypes.xsd)
```

Comment:
maybe this comment should stay? I know I was surprised when I dove into the XML and checking/saving is 1/0 instead of a string but maybe this comment isn't the best way of recording that

Reply:
oh! yeah, this removal of the comment was accidental.